src/modbus/ModbusGroupReader.cpp

```diff
@@ -53,12 +53,13 @@ void ModbusGroupReader::passValuesToGroup(RegisterGroup& group, const std::vecto
         bool newValue = values[i++];
         LOG(TRACE) << "ModbusGroupReader: Old value = " << mapping.second->getBoolValue()
                    << ". New value = " << newValue;
-        if (mapping.second->update(newValue))
+
+        if (mapping.second->doesUpdate(newValue))
```

Comment:
If we just check here and the deadband is not reached then the current value of registers will not be saved.
If we save values by calling the update instead and the deadband is not reached then we risk not reporting small but constant rise (or fall) of values just below the deadband.
What is expected behaviour?

You should probably call doesUpdate as a part of update but handle small but constant changes